Xamarin.PropertyEditing.Mac/Controls/Custom/CommandMenuItem.cs

```diff
@@ -24,16 +25,21 @@ public CommandMenuItem (string title) : base (title)
 			HookUpCommandEvents ();
 		}
 
-		public CommandMenuItem (string title, ICommand command) : this (title)
+		public CommandMenuItem (string title, ICommand command, NSView senderView) : this (title)
 		{
 			this.Command = command;
+			this.senderView = senderView;
 		}
 
 		private void HookUpCommandEvents ()
 		{
 			Activated += (object sender, EventArgs e) => {
-				if (this.command != null)
-					this.command.Execute (null);
+				if (this.command != null) {
+					if (this.command.GetType ().IsGenericType)
+						this.command.Execute (senderView);
```

Comment:
Why would we pass UI elements to commands?

Reply:
I'm struggling now to remember why I made that change :/ I think it was because I had issues losing the view's handle but I'm handling things differently now and it was a hack to get things working, but isn't needed anymore.